resources/kiali/README.md

```diff
@@ -0,0 +1,35 @@
+# Kiali
+
+[Kiali](http://kiali.io) is a spyglass for the istio service mesh and provides answers to the questions:
+What microservices are part of my Istio service mesh? How are they connected? How are they performing?
+
+## Introduction
+
+This chart installs Kiali on a Kyma cluster.
+
+## Prerequisites
+
+- The `monitoring` chart must be installed in order to have Kiali functional
+- The `jaeger` chart should be installed to have the Jaeger integration of Kiali functional
```

Comment:
```suggestion
- Install the Jaeger chart to integrate Jaeger with Kiali. 
```
Can we simply say `integrate` here or `to make Jaeger integration with Kiali work` sounds better? I would suggest the shorter version, but I don't want to change the meaning :)